llama-index-integrations/callbacks/llama-index-callbacks-langfuse/pyproject.toml

```diff
@@ -0,0 +1,64 @@
+[build-system]
+build-backend = "poetry.core.masonry.api"
+requires = ["poetry-core"]
+
+[tool.codespell]
+check-filenames = true
+check-hidden = true
+ignore-words-list = "Gere"
+skip = "*.csv,*.html,*.json,*.jsonl,*.pdf,*.txt,*.ipynb"
+
+[tool.llamahub]
+contains_example = false
+import_path = "llama_index.callbacks.langfuse"
+
+[tool.llamahub.class_authors]
+LangfuseCallbackHandler = "llama-index"
+
+[tool.mypy]
+disallow_untyped_defs = true
+exclude = ["_static", "build", "examples", "notebooks", "venv"]
+ignore_missing_imports = true
+python_version = "3.8"
+
+[tool.poetry]
+authors = ["Your Name <you@example.com>"]
+description = "llama-index callbacks langfuse integration"
+exclude = ["**/BUILD"]
+license = "MIT"
+name = "llama-index-callbacks-langfuse"
+readme = "README.md"
+version = "0.1.2"
+
+[tool.poetry.dependencies]
+python = ">=3.8.1,<4.0"
+llama-index-core = "^0.10.8"
+langfuse = "^2.16.2"
```

Comment:
Should we change this to "^2" so that we automatically upgrade to each 2.x.x version? We should add to our readme.md in the python package that we also need to upgrade here in case we need to.